tools/stats/scribe.py

```diff
@@ -0,0 +1,33 @@
+import os
+import json
+
+import requests  # type: ignore[import]
+import boto3  # type: ignore[import]
+
+
+def send_to_scribe(logs: str) -> str:
+    access_token = os.environ.get("SCRIBE_GRAPHQL_ACCESS_TOKEN", "")
```

Comment:
do we still need to send over http if we have boto3 route?

Reply:
yes, currently circleci are not running in the aws runners - and only those machines have the IAM roles setup

Reply:
i see. good to make some comment on the 2 code path to explain the usage are split between circle and GHA

Reply:
updated